TaskForge.NET/TaskForge.WebUI/wwwroot/js/project-dashboard/task-edit.js

```
@@ -0,0 +1,271 @@
+    });
+}
+
+
+$('input[name="Attachments"]').on('change', function () {
+    const files = this.files;
+    let error = '';
+
+    if (files.length > 10) {
+        error = "You can't upload more than 10 files.";
+    } else {
+        for (let file of files) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'file' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)